source/documentation/06-api-reference.md

```diff
@@ -140,7 +140,7 @@ These error codes provide more information about why a request failed.
 
 The API error codes in this section are driven by payment status. You can see which payments have these error codes using either of the following methods:
 
-- log into the [GOV.UK Pay admin site](https://selfservice.payments.service.gov.uk/), go to _Transactions_ and click on a failed payment
+- log into the [GOV.UK Pay admin site](https://selfservice.payments.service.gov.uk/), go to _Transactions_ and click on a  payment with a "failed", "cancelled" or "error" status.
```

Comment:
Really picky: can you use “curly quotes” here?

Also, there appears to be two spaces between “a” and “payment”.

Reply:
As discussed, it seems that magic in the build process converts "straight quotes" into “curly quotes”, so we’re OK there